src/components/Main/Schedule/Table.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useTable } from 'react-table';
+
+// Components
+import { EditableCell } from './EditableCell';
+
+// Material-UI Components
+import MaUTable from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+// Types
+import { TableProps } from '../../../shared/types';
+
+export const Table = (props: TableProps) => {
+  const { columns, data, updateMyData, title } = props;
+  const { getTableProps, headerGroups, rows, prepareRow } = useTable({
+    columns,
+    data,
+    defaultColumn: { Cell: EditableCell },
```

Comment:
I'm guessing it's supposed to be capital C to match `cell.render('Cell')`? 
I'm not familiar with the react-table API. 
Otherwise: 
```suggestion
    defaultColumn: { cell: EditableCell },
```

Reply:
Yeah, it's part of the api.
https://react-table.tanstack.com/docs/api/useTable

Reply:
Sorry I didn't answer this properly. It has to be `{ Cell: EditableCell }` because Cell is required by the api in capital form. If you look at the docs under column, it's capital C.